aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -862,6 +862,23 @@ Tensor max(const Tensor& self, const Tensor& other) {
   return at::maximum(self, other);
 }
 
+Tensor& fmax_out(Tensor& result, const Tensor& self, const Tensor& other) {
```

Comment:
The result tensor must be the last parameter (see comments on native_functions.yaml)

Reply:
Addressed!